### PR TITLE
Get ref name in shared deploy workflow

### PR DESCRIPTION
**Pull request by joaofbvieira**
Get actions repository reference automatically.
Full credits to @emanuelgaspar.

**STILL BEING TESTED**

### CODE REVIEW

**Review by emanuelgaspar — DISMISSED** [review]
Hope it works 😅

File: .github/workflows/shared-build-deploy-container.yml [hunk 1]
```diff
@@ -43,25 +43,65 @@ on:
...
   GITHUB_TOKEN: ${{ secrets.TRAKX_GITHUB_TOKEN || secrets.GITHUB_TOKEN }}
 
 jobs:
+  # dump_contexts_to_log:
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - name: Dump GitHub context
+  #       env:
+  #         GITHUB_CONTEXT: ${{ toJson(github) }}
+  #       run: echo "$GITHUB_CONTEXT"
```
Comment (emanuelgaspar) [inline .github/workflows/shared-build-deploy-container.yml:58]:
```suggestion
  dump_contexts_to_log:
    runs-on: ubuntu-latest
    steps:
      - name: Dump GitHub context
        run: |
          echo "$GITHUB_ACTION_PATH"
          echo "${{ github.action_path }}"
  #    - name: Dump GitHub context
  #       env:
  #         GITHUB_CONTEXT: ${{ toJson(github) }}
  #       run: echo "$GITHUB_CONTEXT"
```

File: .github/workflows/shared-build-deploy-container.yml [hunk 2]
```diff
@@ -43,25 +43,69 @@ on:
...
   GITHUB_TOKEN: ${{ secrets.TRAKX_GITHUB_TOKEN || secrets.GITHUB_TOKEN }}
 
 jobs:
+  dump_contexts_to_log:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Dump GitHub context
+        run: |
+          echo "$GITHUB_ACTION_PATH"
+          echo "${{ github.action_path }}"
```
Comment (emanuelgaspar) [inline .github/workflows/shared-build-deploy-container.yml:58]:
```suggestion
        env:
          actionPath: ${{env.GITHUB_ACTION_PATH}}
        run: |
          echo "${{env.actionPath}}"
          echo "${{github.action_path}}"
```

**Comment by joaofbvieira** [comment]
Unfortunately it doesn't work 😞

**Pull request closed without merging**